src/Raven.Client/ServerWide/Expiration/ExpirationConfiguration.cs

```diff
@@ -4,15 +4,15 @@ namespace Raven.Client.ServerWide.Expiration
 {
     public class ExpirationConfiguration : IDynamicJson
     {
-        public bool Active { get; set; }
```

Comment:
You have now flipped the meaning here, did you verify that this means the same.
Before, it was off by default, now it is on by default.